doc/source/tune-distributed.rst

```diff
@@ -0,0 +1,31 @@
+How To: Distributed Tuning
+==========================
+
+This is an example walkthrough for a distributed hyperparameter search in Tune. At a high level, we will use Ray's cluster launcher/autoscaler utility to start a cluster of nodes, all of which are connected by Ray. Then, we will modify an existing hyperparameter tuning script to connect to the Ray cluster, and launch the script. Finally, we will analyze the results.
```

Comment:
Remove comma before "and launch the script"

Reply:
I think this would be more readable as a list, e.g.:

This is an example walkthrough for distributed hyperparameter search in Tune, consisting of the following steps:
1. Start a Ray cluster of autoscaling nodes using built-in Ray utilities.
2. Modify and launch an existing hyperparameter tuning script to connect to the Ray cluster.
3. Analyze the results.